lib/presentation/login/view/login_page_view.dart

```diff
@@ -24,53 +24,17 @@ class _LoginPageState extends State<LoginPage> {
 
   @override
   Widget build(BuildContext context) {
+    final welcomeText = AppLocalizations.of(context)!.welcome;
```

Comment:
do we need these variables?

Reply:
I think we do because we pass them for in the AppBar constructor with correct localised text for. that screen because text is different for login and registration screen

Reply:
I can not agree with you. We will need variables only if we are using them on more than one place, but here we need these values only on one place, to forward them through the constructor